AdyenNetworking/APIClient/APIClient.swift

```diff
@@ -57,19 +57,27 @@ public final class APIClient: APIClientProtocol, AsyncAPIClientProtocol {
     /// :nodoc:
     private let urlSession: URLSession
     
+    /// Encoding and Decoding
+    private let coder: AnyCoder
+
     /// Initializes the API client.
     ///
     /// - Parameters:
     ///   - apiContext: The API context.
     ///   - configuration: An optional `URLSessionConfiguration` to be used.
     ///   If no value is provided - `URLSessionConfiguration.ephemereal` will be used.
-    public init(apiContext: AnyAPIContext, configuration: URLSessionConfiguration? = nil) {
+    public init(
+        apiContext: AnyAPIContext,
+        configuration: URLSessionConfiguration? = nil,
+        coder: AnyCoder = Coder()
```

Comment:
Missing meta-dock for  `coder ` parameter.

Reply:
Can you be more specific about the meaning of meta-dock here? Checked readme but could not find relevant information.